ui/DesignSystem/Component/PeerSelector.svelte

```diff
@@ -82,7 +85,9 @@
 
   .peer-dropdown {
     border: 1px solid transparent;
-    border-radius: 0.5rem;
+    border-right: none;
+    border-top-left-radius: 0.5rem;
+    border-bottom-left-radius: 0.5rem;
```

Comment:
```suggestion
    border-bottom-left-radius: 0.5rem;
    border-bottom-right-radius: 0.5rem;
```

Reply:
Huh, that's wrong. I don't understand

Reply:
It looks weird with the bottom right radius being a corner. Wondering if we should just keep the dropdown fully rounded just to keep it consistent with all dropdowns.

Reply:
It does? For me it's rounded... Did you run the code? I round the bottom right corner in the <Entry /> to make sure it's only rounded if there's more then 1 item in the list.

Reply:
Aah, I get it now. It was hard to see in dark mode. I made it dependent on the height of the dropdown, which means that if there is more then 1 item in the list it will be rounded otherwise it's a corner.
https://github.com/radicle-dev/radicle-upstream/pull/1940/commits/f2fd1adb4a2c9774bdca3889aa37badd7f115b3f